gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/GhfsStreamStats.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright 2023 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.hadoop.fs.gcs;
+
+import javax.annotation.Nonnull;
+
+class GhfsStreamStats {
```

Comment:
I know we did not do this in 2.2. But a comment explaining what this class is doing will be useful.

Reply:
Done